recipes/keras-core/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set name = "keras-core" %}
```

Comment:
B/c we use `-core` sometimes to split packages it would be nice to add a comment here mentioning that this package, `keras-core`, is not related to the `keras ` but it is its own thing.

Reply:
See three lines comment, thx